packages/config/config/devices/0x0345/templates/swidget_template.json

```
@@ -0,0 +1,1504 @@
+{
+	"power_1_high_entry_value": {
+		// param 1
+		"label": "Power 1 High Entry Value",
+		"description": "Used to set the power value for Power 1 (the controlled load) that indicates entry into a Power 1 Low power state.",
+		"unit": "W",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 2400,
+		"unsigned": true,
+		"defaultValue": 2400
+	},
+	"power_1_low_entry_value": {
+		// param 2
+		"label": "Power 1 Low Entry Value",
+		"description": "Used to set the power value for Power 1 (the controlled load) that indicates exit from a Power 1 High state.",
+		"unit": "W",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 2400,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_high_exit_value": {
+		// param 3
+		"label": "Power 1 High Exit Value",
+		"description": "Used to set the power value for Power 1 (the controlled load) that indicates exit from a Power 1 High state.",
+		"unit": "W",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 2400,
+		"unsigned": true,
+		"defaultValue": 2400
+	},
+	"power_1_low_exit_value": {
+		// param 4
+		"label": "Power 1 Low Exit Value",
+		"description": "Used to set the power value for Power 1 (the controlled load) that indicates exit from a Power 1 Low power state.",
+		"unit": "W",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 2400,
+		"unsigned": true,
+		"defaultValue": 0
+	},
+	"power_1_high_delay": {
+		//param 5
+		"label": "Power 1 High Delay",
+		"description": "Used to set the time duration that Power 1 High state change power level conditions must exist before Power 1 High state entry and/or exit is registered, and the associated actions are executed.",
```

I think this can be shortened a bit
```suggestion
		"description": "How long the load power must be above/below the thresholds before entering/exiting high power state.",
```